### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by MoAzza98**

### CODE REVIEW

**Comment by MoAzza98** [comment]
Fixed an error that stopped dashes from working. Current build is the one with implemented camera shake on dash.

File: projectsinia/Assets/Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -209,18 +210,17 @@ private void Update()
 		#region DASH CHECKS
 		if (CanDash() && LastPressedDashTime > 0)
 		{
-			CinemachineShake.Instance.ShakeCamera(5f, 0.5f);
+			CinemachineShake.Instance.ShakeCamera(10f, 0.2f);
```
Comment (SimonHolzknecht) [inline projectsinia/Assets/Assets/Scripts/PlayerMovement.cs:213]: nice change!

File: projectsinia/Assets/Assets/Scripts/PlayerMovement.cs [hunk 2]
```diff
@@ -9,6 +9,7 @@ public class PlayerMovement : MonoBehaviour
 
 	public MovementInfoScriptableObject Data;
 	public PlayerAnimator animController;
+	public DashTrail dt;
```
Comment (SimonHolzknecht) [inline projectsinia/Assets/Assets/Scripts/PlayerMovement.cs:12]: Ideally, you want to give variables self-explanatory names. If you write out this short name here for example, you will have an easier time knowing the variables purpose upon next code change

**Review by SimonHolzknecht — COMMENTED** [review]
Just some thoughts on you code

**Pull request merged**